clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1613,7 +1613,8 @@ public Map<TopicPartition, OffsetAndTimestamp> offsetsForTimes(Map<TopicPartitio
      * @see #seekToBeginning(Collection)
      *
      * @param partitions the partitions to get the earliest offsets.
-     * @return The earliest available offsets for the given partitions
+     * @return The earliest available offsets for the given partitions. If the offset for a specific partition cannot be 
+     *         found or the timeout is zero, the corresponding value will be {@code null}
```

Comment:
What do you think of this?

```suggestion
     * @return The earliest available offsets for the given partitions. If the offset for a specific partition cannot be 
     *         found within the default timeout, its entry in the returned map will be {@code null}
```